dali/pipeline/operators/decoder/host_decoder.h

```diff
@@ -59,6 +60,11 @@ class HostDecoder : public Operator<CPUBackend> {
                               input.size(),
                               output_type_,
                               output));
+    } else if (CheckIsTiff(input.data<unsigned char>())) {
```

Comment:
PNG and TIFF cases are the same as fallback to OpenCV, so maybe we should just remove them and leave only two cases - jpeg and all remaining one.

Reply:
OK, I see that is not that easy now. Let postpone it now.